programs/cryptid/src/error.rs

```diff
@@ -49,4 +49,7 @@ pub enum CryptidError {
     /// The accounts passed to execute do not match those in the transaction account.
     #[msg("The accounts passed to execute do not match those in the transaction account.")]
     InvalidAccounts,
+    /// An invalid Middleware Account was passed.
+    #[msg("Invalid Middleware Account.")]
```

Comment:
Updated, but this specific error is Intentionally vague, since the requirements of what a valid Middleware Account is COULD change in the future. right now it just means not an EOA.